lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1990,6 +1990,156 @@ static bool IsSerializedAST(const swift::ModuleDecl &module) {
   });
 }
 
+/// Scan a newly added lldb::Module for Swift modules and report any errors in
+/// its module SwiftASTContext to Target.
+static void
+ProcessModule(Module &module, std::string m_description,
```

Comment:
Nit: rename this from `m_description` to `description` or make this a member function and not pass it as a parameter at all.

Reply:
No, this is a hidden dependency of the LLDB_PRINTF() macro in this file.